edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Edgelet.Test/ModuleManagementHttpClientTest.cs

```diff
@@ -271,6 +271,23 @@ public async Task ModuleLogsTest(string serverApiVersion, string clientApiVersio
             Assert.Equal(buffer.Length, bytesRead);
         }
 
+        [Theory]
+        [MemberData(nameof(VersionMap))]
+        public async Task ModuleLogsTestInvalidDateTime(string serverApiVersion, string clientApiVersion)
+        {
+            // Arrange
+            IModuleManager client = new ModuleManagementHttpClient(this.serverUrl, serverApiVersion, clientApiVersion);
+            var since = "2021-11-01T12:07:0Z";
+            var until = "2021-11-03T12:07:0Z";
```

Comment:
Don't you need two tests? One for since and one for until? Otherwise, this test will always throw on since (because we check since before until) and until will not be covered.

Reply:
@damonbarry : I added test for for both cases along with some happy-path test